app/scenes/channel/channel_dropdown/channel_dropdown_container.js

```diff
@@ -0,0 +1,3 @@
+import ChannelDropdown from './channel_dropdown';
```

Comment:
What I did, is a set in my IDE the insertion of this as part of the new file creation, but those could be fix later on if we need to